.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+
+    - name: Test with pytest
+      run: |
+        pytest --verbose
+
+  security:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)